src/content/guides/integrating-third-party-asset.mdx

```diff
@@ -168,36 +168,36 @@ It is the responsibility of the Asset Integrator to settle funds with a merchant
-| status         | String                                                          | required  | The status of the asset transaction. See [possible status values](#statuses).                            |
-| type           | String                                                          | required  | The type of transaction. Possible values are payment or refund.                                          |
-| failureReason  | String                                                          | optional  | Required if the status is failed. See [possible failure reasons](#failure-reasons).                      |
-| refundable     | Boolean                                                         | optional  | Required if type is payment and status is successful. A flag indicating whether a payment is refundable. |
-| refundBefore   | [Timestamp](/api/data-types#timestamp) | optional  | The latest time at which a refund can be initiated.                                                      |
+| Name           | Type                                                            | Necessity  | Description                                                                                              |
+| -------------- | --------------------------------------------------------------- | ---------- | -------------------------------------------------------------------------------------------------------- |
+| currency       | String                                                          | required   | The three letter [ISO currency code](https://en.wikipedia.org/wiki/ISO_4217) for the payment.            |
+| amount         | String                                                          | required   | The value required to pay in the smallest denomination for the supported currency (e.g. cents).          |
+| authorization  | String                                                          | required   | A reference to the asset of the party paying the Payment Request.                                        |
+| merchantName   | String                                                          | required   | The name of the merchant who created the Payment Request.                                                |
+| merchantId     | String                                                          | required   | Your identifier for the merchant receiving payment.                                                      |
+| idempotencyKey | String                                                          | required   | A unique value that can be used to recognize subsequent retries of the same request.                     |
+| transactionId  | String                                                          | deprecated | A unique ID for the transaction in Centrapay’s system. Also used for idempotency                         |
```

Comment:
@alexBinns Change: Add `deprecated` to `idempotencyKey` and move `idempotencyKey` to the bottom of the list